scripts/sct_compute_mtr.py

```diff
@@ -141,6 +155,12 @@ def get_parser():
                       mandatory=False,
                       example=['0', '1', '2'],
                       default_value='1')
+    parser.add_option(name="-omtr",
+                      type_value="str",
+                      description="Creates output file with the specified path.",
```

Comment:
i would specify to add the extension

Reply:
agreed, fixed in commit [d3d4c22](https://github.com/YangHee-Min/spinalcordtoolbox/commit/d3d4c2292bee56e8d76e3fb201c3ec3ea770c0f5). 
However, is the extension ".nii.gz" the only one possible? If not, the description will be slightly modified.

Reply:
it is not. It could also be `.nii`

Reply:
Understood, fixed in commit [10da873](https://github.com/YangHee-Min/spinalcordtoolbox/commit/10da873575e77cd1af81520931e3769ccb57c5fa)